slice/Ice/Context.ice

```diff
@@ -11,16 +11,15 @@
 [["cpp:include:map"]]
 [["cpp:include:string"]]
 
+[["java:package:com.zeroc"]]
 [["js:module:@zeroc/ice"]]
-
 [["python:pkgdir:Ice"]]
 
-[["java:package:com.zeroc"]]
-
 module Ice
 {
-    /// A string-string dictionary, used to transmit additional information with an Ice request. This context is only
-    /// transmitted with requests, from clients to servers; the Ice protocol does not provide response contexts.
+    /// Represents optional information carried by all Ice requests. This information is filled-in by the application
```

Comment:
While all request carry a context, each request has its context. I would use `an` instead of `all`.

```suggestion
    /// Represents optional information carried by an Ice requests. This information is filled-in by the application
```

Reply:
Fixed.